### PR TITLE
Adding documentation for Aggregates with OVER clause (new feature)

### PR DESCRIPTION
**Pull request by hydersm**

### CODE REVIEW

**Comment by PRMerger10** [comment]
@hysm : Thanks for your contribution! The author(s) have been notified to review your proposed change. @hydersm

File: stream-analytics-docs/over-azure-stream-analytics.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: "OVER (Azure Stream Analytics) | Microsoft Docs"
+description: "Defines the grouping of rows before an associated aggregate or analytic function is applied."
+applies_to: 
+  - "Azure"
+services: stream-analytics
+author: hydersm
+manager: kfile
```
Comment (mamccrea) [inline stream-analytics-docs/over-azure-stream-analytics.md]: remove manager tag

File: stream-analytics-docs/over-azure-stream-analytics.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+ms.service: stream-analytics
+ms.topic: reference
+ms.assetid: 6ae879ff-435d-44e3-bbc7-09cbd399a502
```
Comment (mamccrea) [inline stream-analytics-docs/over-azure-stream-analytics.md]: remove assetid

File: stream-analytics-docs/over-azure-stream-analytics.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+caps.latest.revision: 11
+ms.workload: data-services
+ms.date: 06/18/2019
```
Comment (mamccrea) [inline stream-analytics-docs/over-azure-stream-analytics.md]: update date

**Comment by jborsecnik** [comment]
@mamccrea, are you approving this, or is it awaiting further input? Would you add a [hash comment](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-pull-request-automation?branch=master) to indicate the action to take on this PR? Thanks.

**Comment by mamccrea** [comment]
#please-close

@jborsecnik - this is an internal contribution. we decided not to continue with this PR.

**Pull request closed without merging**